### PR TITLE
docs: include mfa session option for ssh access control

### PR DESCRIPTION
**Pull request by stevenGravy**

### CODE REVIEW

File: docs/pages/server-access/rbac.mdx [hunk 1]
```diff
@@ -178,6 +178,8 @@ spec:
       # Optional: Session recording mode for SSH sessions.
       # If not set, the value set on default will be used.
       ssh: best_effort|strict
+    # require_session_mfa require per-session MFA for any owner of this role
```
Comment (alexfornuto) [inline docs/pages/server-access/rbac.mdx:181]: Is it optional? Is there a default value? Also:

```suggestion
    # require_session_mfa requires per-session MFA for any owner of this role
```

Comment (stevenGravy) [reply docs/pages/server-access/rbac.mdx]: Yes, it's optional, please see update

File: docs/pages/server-access/rbac.mdx [hunk 2]
```diff
@@ -178,6 +178,9 @@ spec:
       # Optional: Session recording mode for SSH sessions.
       # If not set, the value set on default will be used.
       ssh: best_effort|strict
+    # require_session_mfa requires per-session MFA for any owner of this role.
```
Comment (alexfornuto) [inline docs/pages/server-access/rbac.mdx:181]: Is "owner" the right term here? Are you referring to users to whom this role is applied?

Comment (stevenGravy) [reply docs/pages/server-access/rbac.mdx:181]: That's the same language copied from https://goteleport.com/docs/access-controls/reference/.  Basically who's assigned the role.  We can change in both places assigned is better.

File: docs/pages/server-access/rbac.mdx [hunk 3]
```diff
@@ -178,6 +178,9 @@ spec:
...
+    # Optional: the default is false.
```
Comment (zmb3) [inline docs/pages/server-access/rbac.mdx:182]:
```suggestion
    # Require an additional MFA tap to start new sessions.
    # Optional: the default is false.
```

**Review by alexfornuto — APPROVED** [review]

**Review by lsgunn-teleport — APPROVED** [review]
LGTM

**Pull request merged**